CMakeLists.txt

```diff
@@ -48,52 +48,27 @@ if (NOT MSVC)
    set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} ${GCC_COMPILE_FLAGS} -D_DEBUG")
 endif()
 
-set(BASISU_SRC_LIST ${COMMON_SRC_LIST} 
-	basisu_backend.cpp
-	basisu_basis_file.cpp
-	basisu_comp.cpp
-	basisu_enc.cpp
-	basisu_etc.cpp
-	basisu_frontend.cpp
-	basisu_global_selector_palette_helpers.cpp
-	basisu_gpu_texture.cpp
-	basisu_pvrtc1_4.cpp
-	basisu_resampler.cpp
-	basisu_resample_filters.cpp
-	basisu_ssim.cpp
-	basisu_tool.cpp
-	basisu_astc_decomp.cpp
-	basisu_uastc_enc.cpp
-	basisu_bc7enc.cpp
-	lodepng.cpp
-	apg_bmp.c
-	jpgd.cpp
-	transcoder/basisu_transcoder.cpp
-	)
-
 if (APPLE)
    set(BIN_DIRECTORY "bin_osx")
 else()
    set(BIN_DIRECTORY "bin")
 endif()
-
 set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/${BIN_DIRECTORY})
 
-add_executable(basisu ${BASISU_SRC_LIST})
```

Comment:
What happened to this executable?

Reply:
I renamed it `basisu_tool`  but i rollback this change. `basisu` is now on tool/CMakeLists.txt (https://github.com/BinomialLLC/basis_universal/pull/166/files#diff-0c95bbdc6108c42a4a2667b31c0df6eaR3)